css/css-scroll-snap/scroll-target-align-003.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and snap position with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#choosing'>
```

Comment:
I agree that's the desired behavior, but where in the spec does it say that this is supposed to work? The spec is explicit about `:target` and `scrollIntoView()`, but not about `focus()`. Is it already using some anchoring terminology that `focus()` falls under that I just failed to notice? If not, we should either do that, or list focus() explicitly along with the rest.

Reply:
focus() implies a scroll-into-view operations in most UAs that I know, that implied operation should be affected the same way as an explicit one.

Reply:
https://html.spec.whatwg.org/multipage/interaction.html#dom-focus invokes scroll into view, FWIW. (As of https://github.com/whatwg/html/pull/2787 )